pkg/api/validation/validation.go

```diff
@@ -1934,6 +1935,16 @@ func ValidateSeccompPodAnnotations(annotations map[string]string, fldPath *field
 	return allErrs
 }
 
+func validateAppArmorProfile(p string, fldPath *field.Path) field.ErrorList {
+	if p == "docker/default" {
```

Comment:
Thinking about this more, I think sticking with `runtime/default` is the way to go. If a user really wants to enforce the default docker profile, they can do so with `local/docker-default`.